includes/html/print-interface.inc.php

```diff
@@ -72,7 +72,11 @@
 
 echo '</span>';
 
+$port_groups = [0 => 'Default'] + dbFetchKeyValue('SELECT id, name from port_groups order by name');
+
 echo "</td><td width=100 onclick=\"location.href='" . generate_port_url($port) . "'\" >";
+echo $port_groups[$port['port_group_id']];
```

Comment:
why do you fetch all port groups if you only need one?

Reply:
reduced to searched value